public/styles/globals.scss

```diff
@@ -1,6 +1,7 @@
 @import 'bootstrap/dist/css/bootstrap.min.css';
-@import './colorscheme-oceanside.min.css';
-@import './cagov.font-only.css';
+@import './cagov.core.css';
+@import './colorscheme-oceanside.css';
+@import './custom.css';
```

Comment:
Note: these are the substantive changes to this file. Other changes are auto-lint changes.